examples/snippets/modules/ROOT/examples/base-images-override/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version = 1.4.4
```

Comment:
the rest of the repo standardize on `1.4.9`

Reply:
The vast majority of the examples are using 1.4.4, and this was probably just following one of them. I thought about changing all of them to use `1.4.9`, but this PR already has too many changes (this is why I've started https://github.com/lightbend/cloudflow/pull/1075).

Reply:
fine with it